api/commands/build.ts

```diff
@@ -88,7 +89,20 @@ export class Builder {
 
     const tasks = table.disabled
       ? ([] as dataform.IExecutionTask[])
-      : this.adapter.publishTasks(table, this.runConfig, tableMetadata).build();
+      : this.adapter
+          .publishTasks(
+            table,
+            {
+              ...this.runConfig,
```

Comment:
sorry, i think i wasn't clear enough. why not create `runConfig` *once*, in `build()`, before it's used anywhere, including both the overrides for `useRunCache` and `useSingleQueryPerAction`? then just set the `ExecutionGraph.runConfig` to that single `runConfig` (instead of repeatedly overriding the `runConfig`.

This would also be nicer because then `useSingleQueryPerAction` would be logged as part of the run (whereas right now it would not)

Reply:
Ah I see, have done now